lib/foreman_theme_satellite/documentation.rb

```diff
@@ -77,28 +77,38 @@ module Documentation
       'foreman_discovery' => "#{ForemanThemeSatellite.documentation_root}/provisioning_hosts/discovering-hosts-on-a-network_provisioning",
     }.freeze
 
-    DOCS_GUIDES_LINKS = {
-      'Managing_Content' => {
-        'Products_and_Repositories_content-management' => "#{ForemanThemeSatellite.documentation_root}/managing_content/importing_content_content-management#Products_and_Repositories_content-management",
-      },
+    # Guide mapping. If no entry is present, it should be assumed that it can
+    # be downcased
+    DOCS_GUIDE_MAPPING = {
```

Comment:
It will conflict with this [change](https://github.com/RedHatSatellite/foreman_theme_satellite/pull/45/files#diff-03deddd1ea19da8194b5ae0d27b67e7804f3618c2c8ed058c78a3a8ccbf5d622R83), I suggest we rebase once the PR is merged